### PR TITLE
ETH cleaners and fixes

### PR DESCRIPTION
**Pull request by peachbits**
Improvements
-Replace schema with cleaners for transaction history api calls
-Only calculate 'data' parameter if using default fees
-Add cloudflare rpcServer

Bug fixes:
-Fix TRANSACTION_STORE_FILE data initialization
-Fix hex number parsing
-Pass fetchCors function to amberdata api calls
-Remove unnecessary log

Dependency updates: 
-Add postinstall script for node14 dependency compatibility (usb and node-hid)
-Update cleaners

### CODE REVIEW

File: src/ethereum/ethNetwork.js [hunk 1]
```diff
@@ -1447,36 +1486,44 @@ export class EthereumNetwork {
...
   }
 
   async checkTokenBal(tk: string): Promise<EthereumNetworkUpdate> {
     return asyncWaterfall([
       async () => this.checkTokenBalEthscan(tk),
       this.checkTokenBalBlockchair
     ]).catch(err => {
-      this.ethEngine.log('checkTokenBal failed to update', err)
+      this.ethEngine.log(
+        'thisisauniquestringcheckTokenBal failed to update',
```
Comment (peachbits) [inline src/ethereum/ethNetwork.js:1524]: remove

File: src/ethereum/ethNetwork.js [hunk 2]
```diff
@@ -507,7 +507,7 @@ export class EthereumNetwork {
       if (!alchemyApiKey || alchemyApiKey.length < 6) {
         throw new Error('Need Alchemy API key')
       }
-      addOnUrl = `/v2/-${alchemyApiKey}`
+      addOnUrl = `/v2/${alchemyApiKey}`
```
[the remark this answers is not in the document]
Comment (peachbits) [reply src/ethereum/ethNetwork.js]: Reverted this change. Not sure how to correct this the right way

File: src/ethereum/ethTypes.js [hunk 3]
```diff
@@ -167,6 +176,100 @@ export type AlethioTokenTransfer = {
...
+      layer_2: asObject({
+        erc_20: asArray(asOptional(asString))
+      })
+    })
+  )
+})
+
+export type CheckTokenBalBlockchair = $Call<typeof asCheckTokenBalBlockchair>
+
 export type AmberdataTx = {|
```
Comment (paullinator) [inline src/ethereum/ethTypes.js:273]: Looks like this can be removed.

File: src/ethereum/ethTypes.js [hunk 4]
```diff
@@ -181,6 +284,53 @@ export type AmberdataTx = {|
...
+  payload: asObject({
+    records: asArray(asEither(asAmberdataAccountsTx, asAmberdataAccountsFuncs))
+  })
+})
+
+export type FetchGetAmberdataApiResponse = $Call<
+  typeof asFetchGetAmberdataApiResponse
+>
+
 export type AmberdataInternalTx = {|
```
Comment (paullinator) [inline src/ethereum/ethTypes.js:334]: Should be able to remove `AmberdataInternalTx`

**Review by paullinator — APPROVED** [review]

**Pull request merged**